x-pack/plugins/secops/.gitattributes

```diff
@@ -3,3 +3,4 @@
 x-pack/plugins/secops/public/graphql/types.ts linguist-generated=true
 x-pack/plugins/secops/server/graphql/types.ts linguist-generated=true
 x-pack/plugins/secops/public/graphql/introspection.json linguist-generated=true
+
```

Comment:
As a PSA, changes to `.gitattributes` usually take a little while to be reflected on Github https://github.com/github/linguist/issues/3737#issuecomment-317981064